tools/build_TizenRT/Makefile

```diff
@@ -64,10 +64,13 @@ CSRCS		= src/libnnstreamer-edge/nnstreamer-edge-data.c \
 
 CFLAGS += -I ./include -DDEBUG=0
 
-# TODO Include sources - Paho MQTT and AITT library.
-# Paho MQTT library
-# CSRCS += nnstreamer-edge-mqtt.c
-# CFLAGS += -DENABLE_MQTT=1
+ifeq ($(CONFIG_NETUTILS_MQTT),y)
+	CFLAGS += -I ../mosquitto
```

Comment:
check this line (unnecessary?)

Reply:
I think this line is necessary.
refer:
https://github.com/Samsung/TizenRT/blob/589e026d2f9467362b346fd490d240e9d9a420a1/framework/src/network/mqtt/Make.defs#L59